js/src/admin/components/AdminNav.js

```diff
@@ -31,76 +31,119 @@ export default class AdminNav extends Component {
           icon: 'far fa-chart-bar',
-          description: app.translator.trans('core.admin.nav.dashboard_text'),
+          title: app.translator.trans('core.admin.nav.dashboard_title'),
         },
         app.translator.trans('core.admin.nav.dashboard_button')
       )
     );
 
     items.add(
       'basics',
-      AdminLinkButton.component(
+      LinkButton.component(
         {
           href: app.route('basics'),
           icon: 'fas fa-pencil-alt',
-          description: app.translator.trans('core.admin.nav.basics_text'),
+          title: app.translator.trans('core.admin.nav.basics_title'),
         },
         app.translator.trans('core.admin.nav.basics_button')
       )
     );
 
     items.add(
       'mail',
-      AdminLinkButton.component(
+      LinkButton.component(
         {
           href: app.route('mail'),
           icon: 'fas fa-envelope',
-          description: app.translator.trans('core.admin.nav.email_text'),
+          title: app.translator.trans('core.admin.nav.email_title'),
         },
         app.translator.trans('core.admin.nav.email_button')
       )
     );
 
     items.add(
       'permissions',
-      AdminLinkButton.component(
+      LinkButton.component(
         {
           href: app.route('permissions'),
           icon: 'fas fa-key',
-          description: app.translator.trans('core.admin.nav.permissions_text'),
+          title: app.translator.trans('core.admin.nav.permissions_title'),
         },
         app.translator.trans('core.admin.nav.permissions_button')
       )
     );
 
     items.add(
       'appearance',
-      AdminLinkButton.component(
+      LinkButton.component(
         {
           href: app.route('appearance'),
           icon: 'fas fa-paint-brush',
-          description: app.translator.trans('core.admin.nav.appearance_text'),
+          title: app.translator.trans('core.admin.nav.appearance_title'),
         },
         app.translator.trans('core.admin.nav.appearance_button')
       )
     );
 
     items.add(
-      'extensions',
-      AdminLinkButton.component(
-        {
-          href: app.route('extensions'),
-          icon: 'fas fa-puzzle-piece',
-          description: app.translator.trans('core.admin.nav.extensions_text'),
-        },
-        app.translator.trans('core.admin.nav.extensions_button')
-      )
+      'search',
+      <div className="Search-input">
+        <input
+          className="FormControl SearchBar"
+          bidi={this.query}
+          type="search"
+          placeholder={app.translator.trans('core.admin.nav.search_placeholder')}
+        />
+      </div>
     );
 
     return items;
   }
+
+  extensionItems() {
+    const items = new ItemList();
+
+    const categorizedExtensions = getCategorizedExtensions();
+    const categories = app.extensionCategories;
+
+    Object.keys(categorizedExtensions).map((category) => {
+      if (!this.query()) {
+        items.add(
+          `${category} NavDivider`,
```

Comment:
I don't know how I feel about having class names for the parent list item in the item name...

Do we do this anywhere else?